content/about/about.en.md

```diff
@@ -11,7 +11,7 @@ scalable network applications. In the following "hello world" example, many
 connections can be handled concurrently. Upon each connection, the callback is
 fired, but if there is no work to be done, Node.js will sleep.
 
-```javascript
+```js
```

Comment:
I think this is an unrelated change to the PR, but I'll allow it ;)

Reply:
It's just to standardize.

Reply:
Yes, but these are unrelated changes. Keep in mind, that we should avoid unrelated changes. To be fair, I also sometimes include unrelated changes when I believe the PR benefits from then, in this case, it's ok.